src/components/DocHolder.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { Colors } from '../../assets/Colors';
+import { TextRegular } from '../../assets/fonts/Fonts';
+import { AntDesign, Feather, FontAwesome } from '@expo/vector-icons';
+
+
+
+interface DocProps {
+    title: string;
+    submitted: boolean;
+    onClick?: () => void;
+    
+}
+
+
+const Upload = <Feather name='upload' />
+
+const Trash = <FontAwesome name='trash-o' style = {{ color: Colors.brandGray }}/>
+
+export const Missing = <AntDesign name='exclamationcircle' style={{ color: Colors.brandRed}} />
+
+export const Submitted = <AntDesign name='checkcircle' style={{ color: Colors.brandBlue }} />
+
+
+const DocSelect = styled.TouchableOpacity`
```

Comment:
Same as above: move to separate file. Maybe group your Doc component into a folder within `src/Components` and move all styled components into 1 styles file.